test-lib.sh

```diff
@@ -95,6 +95,33 @@ function ct_cleanup() {
   exit "${TESTSUITE_RESULT:-0}"
 }
 
+# ct_build_image_and_parse_id
+# --------------------
+# Return 0 if build was successful, 1 otherwise
+# Uses: $1 - path to docckerfile
+# Uses: $2 - build params
+# Uses: $APP_IMAGE_ID - sets the app image id value to this variable
+# this should be replaced by the --iidfile parameter
+# when it becames supported by all versions of podman and docker that we support
+ct_build_image_and_parse_id() {
+  local tmpdir
+  local log_file
+  local ret_val
+  local dockerfile
+  local command
+  log_file="$(mktemp)"
+  ret_val=1
+  [ -n "$1" ] && dockerfile="-f $1"
+  command="$(echo "docker build --no-cache $dockerfile $2" | tr -d "'")"
```

Comment:
What is the propose of `tr -d "'"`?

Reply:
Because when inserting parameters from a variable to the command they were eclosed with '' and the command then did not work.